server/tso/tso.go

```diff
@@ -68,11 +68,26 @@ func (t *timestampOracle) setTSOPhysical(next time.Time) {
 	t.tsoMux.Lock()
 	defer t.tsoMux.Unlock()
 	// make sure the ts won't fall back
-	if t.tsoMux.tso == nil || typeutil.SubTimeByWallClock(next, t.tsoMux.tso.physical) >= updateTimestampGuard {
+	if t.tsoMux.tso == nil || typeutil.SubTimeByWallClock(next, t.tsoMux.tso.physical) > 0 {
 		t.tsoMux.tso = &tsoObject{physical: next}
 	}
 }
 
+func (t *timestampOracle) setTSO(nextPhysical time.Time, nextLogical int64) {
```

Comment:
Do we need to update the persisted time window on etcd?

Reply:
Yes, but this method is only used to set the TSO in memory and the only place we call `setTSO` for now is in `ResetUserTimestamp` which guarantees the time window updating at L221.

Reply:
got it